package.json

```diff
@@ -21,12 +21,8 @@
     "@sentry/webpack-plugin": "^1.20.0",
     "@types/dompurify": "^2.0.3",
     "@types/node": "^12",
-    "@types/react": "^16.9.46",
-    "@types/react-dom": "^16.9.8",
```

Comment:
For some reason these were pinning to v18 of the types, I had to move them to the resolutions for now. I'll revisit later.